packages/dd-trace/src/config.js

```diff
@@ -707,6 +709,7 @@ class Config {
     this._setBoolean(env, 'traceId128BitGenerationEnabled', DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED)
     this._setBoolean(env, 'traceId128BitLoggingEnabled', DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED)
     this._setBoolean(env, 'tracing', DD_TRACING_ENABLED)
+    this._setBoolean(env, 'apmTracingEnabled', DD_APM_TRACING_ENABLED)
```

Comment:
idk if `DD_APM_TRACING_ENABLED` is appropriate (it's the name indicating the RFC) because there is already a `DD_TRACING_ENABLED` var used to enable/disable tracing entirely via config and RC. And RC capability is called `APM_TRACING_ENABLED`.

So, adding `DD_APM_TRACING_ENABLED` could be a bit confusing...

Reply:
The different options I see:
1. we make a new variable (`DD_APM_TRACING_ENABLED`) and we start having very confusing collisions: `DD_TRACE_ENABLED`, `DD_TRACING_ENABLED`, `DD_APM_TRACING_ENABLED` + RC (and future) counterparts. They all do something slightly different with a slightly different name. The risk is that customers will be absolutely lost and use the wrong vars.
2. We use the current `DD_TRACING_ENABLED`, and we make a stealth change that will actually keep APM enabled when using standalone billing. The risk is that some customers might rely on APM being disabled so that's kind of a breaking change...
3. REVOLUTION! We breaking change all the things and use what the spec suggests `DD_TRACE_ENABLED` to disable APM, and that's a huge breaking change.

Choose your poison.
In all cases, APM teams needs to be consulted.

Reply:
`DD_APM_TRACING_ENABLED` changed to `DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED`